.github/workflows/get_build_targets.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+readonly TMP_OUTPUT_FILE="tmp.txt"
+
+# This limits the directory that bazel query is going to search under.
+readonly SEARCH_FOLDER="//source/common/..."
+
+set -e -o pipefail
+
+git fetch https://github.com/envoyproxy/envoy.git master 2>/dev/null
+
+git diff --name-only $ORIGINAL_HEAD FETCH_HEAD | while IFS= read -r line
```

Comment:
This is probably fine, but it could be have been structured as a bash `$(call some function)`, e.g. https://stackoverflow.com/questions/33201873/bash-functions-inside-process-substitution

Reply:
I am not entirely sure which part of this you want me to put it into a function?

Reply:
All of the code that is writing to the temp file.

Reply:
@htuch  should be all set now.